src/Tizen.Multimedia.Metadata/MetadataExtractor/Metadata.cs

```diff
@@ -402,6 +403,16 @@ internal Metadata(MetadataExtractor extractor)
         /// <value>A string representing the information for 360 content, or null if the information does not exist.</value>
         public string Content360 { get; }
 
+        /// <summary>
+        /// Gets the information for stitched 360 content of the media.
+        /// </summary>
+        /// <value>
+        /// 0:None(There's no stitched 360 content metadata), 1:Non-stitched content, 2:stitched content
```

Comment:
is it 0 or None or maybe both for cases when content does not specify that information?
Could we have an enum with the listed possible values? That would make the description easier.  Or maybe nullable int (`int?`) could be used? That would also depend on underlying metadata stored.

Reply:
I'll change it as `int?` type.